src/shared/components/post/post-listing.tsx

```diff
@@ -1010,9 +1010,8 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
     const body = post.body;
 
     return body
-      ? `${I18NextService.i18n.t("cross_posted_from")} ${
-          post.ap_id
-        }\n\n${body.replace(/^/gm, "> ")}`
+      ? `${I18NextService.i18n.t("cross_posted_from", post.ap_id)}
```

Comment:
Use the `@${hostname(community.ap_id)}`;` . 

Also I believe these need objects, ie 

`I18NextService.i18n.t("cross_posted_from", {ap_id: post.ap_id})